rts/Sim/Units/UnitDef.cpp

```diff
@@ -818,7 +818,8 @@ void UnitDef::CreateYardMap(std::string&& yardMapStr)
 			case 'c': { defYardMap[ymCopyIdx - 1] = YARDMAP_YARD;                         } break;
 			case 'i': { defYardMap[ymCopyIdx - 1] = YARDMAP_YARDINV;                      } break;
 			case 'b': { defYardMap[ymCopyIdx - 1] = YARDMAP_BUILDONLY;                    } break;
-//			case 'w': { defYardMap[ymCopyIdx - 1] = YARDMAP_WALKABLE;                     } break; // TODO?
+			case 'u': { defYardMap[ymCopyIdx - 1] = YARDMAP_WALKONLY;                     } break;
```

Comment:
perhaps 'w' for 'walk'?

Reply:
I wanted to, but 'w' was already taken as one of the four letters for a blocked square. I'm not sure what the history is of that. If we can drop the w/x/f letters from blocked, then I'd be happy to use 'w' for walkable.

Reply:
'w' used to mean 'water only' in the TA engine, while 'o' was land only.

Although on second thought maybe having 'water only' and 'ground only' yardmap types would be good for "age of" style docks (generally, coastal buildings that need to be put at the edge of water, such that you can't solve it via min/max height).
![image](https://github.com/beyond-all-reason/spring/assets/2573076/ebaf3299-10d4-4f3d-864e-448ab09d9447)

Perhaps keep 'u' but rename the tile type into YARDMAP_**U**NBUILDABLE?

Reply:
Sounds reasonable. Done.